Kickstarter-iOS/Views/Controllers/CommentsViewController.swift

```diff
@@ -8,16 +8,34 @@ import UIKit
 internal final class CommentsViewController: UITableViewController {
   // MARK: - Properties
 
+  private lazy var commentComposer: CommentComposerView = {
+    let frame = CGRect(x: 0, y: 0, width: view.frame.width, height: 80)
+    let view = CommentComposerView(frame: frame)
+    return view
+  }()
+
   fileprivate let viewModel: CommentsViewModelType = CommentsViewModel()
 
   // MARK: - Lifecycle
 
   internal override func viewDidLoad() {
     super.viewDidLoad()
 
+    tableView.keyboardDismissMode = .interactive
```

Comment:
This is cool! I didn't know this existed to make the keyboard disappear on scroll!